src/Component/src/Metadata/Extractor/PhpFileResourceExtractor.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Metadata\Extractor;
+
+use Sylius\Resource\Metadata\ResourceMetadata;
+
+final class PhpFileResourceExtractor extends AbstractResourceExtractor
+{
+    /**
+     * @inheritdoc
+     */
```

Comment:
I might be mistaken, but I think inheritdoc is an outdated phpdoc and static analyzer will understand it anyway. If it's true, then all instances of this needs to be removed in the PR.

```suggestion
```